.gitignore

```diff
@@ -156,4 +156,6 @@ GitHub.sublime-settings
 *.patch
 
 #Ignore personal notes
-notes.txt
\ No newline at end of file
+notes.txt
+**/node_modules/
+**/*package*.json
```

Comment:
since we are ignoring uploading these, is loading the needed packages added to the makefile?

Reply:
That's just for now to not clutter up the main with unused stuff